tools/docker/daml-all-in-one/README.md

```diff
@@ -8,7 +8,12 @@ An all in one DAML docker image with the `sample ledger contracts`.
 
 To build the daml-all-in-one image locally, use:
 ```sh
-docker build ./tools/docker/daml-all-in-one/ -t daml-all-in-one
+DOCKER_BUILDKIT=1 docker build \
+  --file ./tools/docker/daml-all-in-one/Dockerfile \
+  ./tools/docker/daml-all-in-one/ \
+   --tag daio \
+   --tag daml-all-in-one \
+   --tag ghcr.io/hyperledger/daml-all-in-one:$(date +"%Y-%m-%dT%H-%M-%S" --utc)-dev-$(git rev-parse --short HEAD)
 ```
```

Comment:
@raynatopedrajeta Please also update the GitHub org name to the new one now that we've migrated over:

```suggestion
   --tag ghcr.io/hyperledger-cacti/daml-all-in-one:$(date +"%Y-%m-%dT%H-%M-%S" --utc)-dev-$(git rev-parse --short HEAD)
```